src/gc/gcee.cpp

```diff
@@ -405,7 +404,41 @@ void ProfScanRootsHelper(Object** ppObject, ScanContext *pSC, uint32_t dwFlags)
     }
 #endif //INTERIOR_POINTERS
     ScanRootsHelper(&pObj, pSC, dwFlags);
-#endif //  defined(FEATURE_EVENT_TRACE)
+#endif //  defined(GC_PROFILING) || defined(FEATURE_EVENT_TRACE)
+}
+
+// TODO - at some point we would like to completely decouple profiling
```

Comment:
Sounds good. I'll likely be revisiting this in more detail at some point soon where I can do just that.